charts/rancher-turtles/templates/rancher-turtles-exp-etcdrestore-components.yaml

```
@@ -45,77 +45,51 @@ spec:
             properties:
               clusterName:
                 type: string
-              configRef:
-                type: string
               location:
                 type: string
               machineName:
                 type: string
             required:
             - clusterName
-            - configRef
-            - location
-            - machineName
             type: object
             x-kubernetes-validations:
             - message: ETCD snapshot location can't be empty.
```

```suggestion
            - message: cluster name can't be empty.
```
Shouldn't this be referencing cluster name?

---

This CRD needs redesign, which will be addressed in a separate PR. I’ll update validation message there too.